packages/pass-style/src/deeplyFulfilled.js

```diff
@@ -105,6 +105,13 @@ export const deeplyFulfilled = async val => {
       // @ts-expect-error not assignable to type 'DeeplyAwaited<T>'
       return E.when(Promise.all(valPs), vals => harden(vals));
     }
+    case 'byteArray': {
+      const bytes = /** @type {ByteArray} */ (val);
+      // @ts-expect-error Why
+      // "Type 'ArrayBuffer' is not assignable to type 'DeeplyAwaited<T>'."?
+      // TODO fix.
```

Comment:
Aligning with the other clauses:
```suggestion
      // @ts-expect-error not assignable to type 'DeeplyAwaited<T>'
```